src/app/find/locationDetails/locationDetails.css

```diff
@@ -0,0 +1,90 @@
+@value primaryColor from "../../../colors.css";
+
+.locationLinks{
+	color: primaryColor;
+	border-bottom: 1px solid primaryColor; 
+	font-size: 0.85rem;
+	cursor: pointer;
+	padding: 2vh 0 0 0;
+}
+
+.locationHeaders{
+	font-size: 0.95rem !important;
+    line-height: 1.33rem;
+    margin: 4.5vh 0 0vh 0;
+    padding-top: 1.2vh;
+    width: 100%;
+    border-top: 1px solid #dadada;
+}
+
+.categoryCard{
+	background-color: #f8f8fc;
+	border: 1px solid #dadada !important;
```

Comment:
I know. Thing is I'm plowing through it. I know it's not the most elegant or efficient approach on the long run, but it's probably the one that allows for contributing and shipping fast. Given the volatility the UI is going to have, for now I feel it's a lesser evil.

Reply:
Yeah, I suppose that's probably true for these `!important` flags.
If it's not fairly easy to fix them, I guess it's OK to leave them for now.

In general, though, these kinds of issues don't really hurt in the "long run" - it can take a surprisingly short time for little hacks to add up and severely impact velocity, hampering the speed of shipping and iteration they were meant to serve in the first place (and also making the code brittle, where any change may lead to unexpected bugs).
It's a general rule of thumb for these kinds of little issues that fixing them when they come up is orders of magnitude faster than figuring them out later, when you've lost the context of why they were added in the first place and possibly compounded the problem with further little hacks.
(As a side note, the same can not be said for bigger design issues, where sometimes it _is_ actually more cost-effective to put off solving them, since additional experience or data can help.)